server/src/domain/sql_migrations.rs

```diff
@@ -460,30 +459,81 @@ async fn migrate_to_v3(pool: &DbConnection) -> anyhow::Result<()> {
     Ok(())
 }
 
+async fn migrate_to_v4(pool: &DbConnection) -> anyhow::Result<()> {
```

Comment:
From what i think, this will fail if two users already exist with the same email, correct? It might be worth making note, and making sure this is labelled as a breaking change?

Reply:
Yeah, that's what the `.context` in the body means: it adds some context to the error message, which will show up in the logs.

And yes, it'll be a breaking change.